docs/design/autoware-interfaces/ad-api/list/api/route/lanelet/notice.md

```diff
@@ -0,0 +1,14 @@
+# /api/route/lanelet/notice
```

Comment:
Considering other namings, I feel it's better to rename it.

```
/api/driving/state
/api/driving/engage
/api/interface/version
```

@yukkysaito @mitsudome-r What do you think?

```suggestion
# /api/routing/route
```

Or other ones?

```suggestion
# /api/routing/lanelet_route
```

```suggestion
# /api/route/lanelet_route
```

Reply:
I prefer /api/routing/route since lanelet is just implementation to express route.

Reply:
I prefer /api/routing/route too :+1:

Reply:
@kenji-miyake @mitsudome-r @yukkysaito 

The API under the namespace `lanelet` has lanelet dependent information. So I think it is prefer to specify the implementation.
- `/api/route/set` uses poses for  waypoints
- `/api/route/lanelet/set` uses lanalet segments for waypoint
- `/api/route/lanelet/notice` uses lanalet segments for waypoint

(`/api/route/notice` cannot be provided because waypoint poses are ambiguous when `/api/route/lanelet/set`)

Reply:
I personally feel the following are more descriptive.

- `/api/route/set_goal_pose`
- `/api/route/set_route`
- `/api/route/route`

But considering `route/route` is a bit weird and there is `/api/driving`, using `routing` looks better.

- `/api/routing/set_goal_pose`
- `/api/routing/set_route`
- `/api/routing/route`

Reply:
For example, how about this?

```
/api/routing/set_route -> /api/routing/plan_route
/api/routing/lanelet/set_route -> /api/routing/set_route
/api/routing/lanelet/route -> /api/routing/route
```

Reply:
`set_route_points` looks good to me. :+1:

Reply:
/api/routing/state
/api/routing/route
/api/routing/set_route_points
/api/routing/set_route
/api/routing/clear_route

Reply:
Cool.